### PR TITLE
add(epp): network-aruba-cppm-snmp

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+* Net-Aruba-Cppm-SNMP-custom
+
+It brings the following service templates:
+
+| Service Alias | Service Template                 | Service Description                        | Default | Discovery |
+|:--------------|:---------------------------------|:-------------------------------------------|:--------|:----------|
+| Cpu           | Net-Aruba-Cppm-Cpu-SNMP          | Check cpu                                  | X       |           |
+| Disks         | Net-Aruba-Cppm-Disks-SNMP        | Check disks                                | X       |           |
+| Interfaces    | Net-Aruba-Cppm-Interfaces-SNMP   | Check interfaces                           |         | X         |
+| Memory        | Net-Aruba-Cppm-Memory-SNMP       | Check memories                             | X       |           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md:23]:
```suggestion
| Memory        | Net-Aruba-Cppm-Memory-SNMP       | Check memory                            | X       |           |
```

File: pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your **Aruba CPPM**
+server. Please refer to the official documentation from XXX:
+* LINK
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md:120]: missing link

File: pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your **Aruba CPPM**
+server. Please refer to the official documentation from XXX:
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md:119]: "from xxx"

File: pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new host through **Configuration > Hosts**.
+* Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your **Aruba CPPM** server settings.
+* Apply the **Net-Aruba-Cppm-SNMP-custom** template to the host.
+
+If you are using SNMP Version 3, use the **SNMPEXTRAOPTIONS** macro to configure
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-cppm-snmp.md:209]:
```suggestion
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**